src/main/java/com/rocketden/main/util/Utility.java

```diff
@@ -0,0 +1,33 @@
+package com.rocketden.main.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Utility {
+
+  /**
+   * Convert the Request Body in 'x-www-form-urlencoded' format to a HashMap 
```

Comment:
In the future this can be removed if we decide to change request body format to JSON, in which we can use DTO objects instead of Strings. But I think that's beyond the scope of this PR :)

Reply:
Yes I think you're likely right  -- I'll look into this, and if `application/json` seems better, I'll implement that in a separate CL.